test/src/main/java/org/apache/accumulo/test/CloseScannerIT.java

```diff
@@ -58,7 +58,7 @@ public void testManyScans() throws Exception {
 
           for (int j = 0; j < i % 7 + 1; j++) {
             // only read a little data and quit, this should leave a session open on the tserver
-            Iterators.get(scanner.iterator(), 10);
+            var unusedRetVal = Iterators.get(scanner.iterator(), 10);
```

Comment:
An assertNotNull() or similar around this would probably suffice.

Reply:
Wrapped call with assertNotNull to remove flagged EP error.